Doc/using/mac.rst

```diff
@@ -2,190 +2,433 @@
-are completely new to Python you should start reading the tutorial introduction
-in that document.
-
-If you are familiar with Python on other Unix platforms you should read the
-section on running Python scripts from the Unix shell.
-
+   Recent versions of macOS include a :command:`python3` command in :file:`/usr/bin/python3`
+   that links to a usually older and incomplete version of Python provided by and for use by
+   the Apple development tools, :program:`Xcode` or the :program:`Command Line Tools for Xcode`.
+   You should never modify or attempt to delete this installation, as it is
+   Apple-controlled and is used by Apple- or third-party software.  If
+   you choose to install a newer Python version from python.org, you will have
+   two different but functional Python installations on your computer that
+   can co-exist. The default installer options should ensure that its :command:`python3`
+   will be used instead of the system :command:`python3`.
 
 How to run a Python script
 --------------------------
 
-Your best way to get started with Python on macOS is through the IDLE
-integrated development environment; see section :ref:`ide` and use the Help menu
-when the IDE is running.
+There are two ways to invoke the Python interpreter.
+If you are familiar with using a Unix shell in a terminal
+window, you can just invoke |python_x_dot_y_literal| or ``python3`` optionally
+followed by one or more command line options (described in :ref:`using-on-general`).
+The Python tutorial also has a useful section on
+:ref:`using Python interactively from a shell <tut-interac>`.
+
+You can also invoke the interpreter through an integrated
+development environment.
+:ref:`idle` is a basic editor and interpreter environment
+which is included with the standard distribution of Python.
+:program:`IDLE` includes a Help menu that allows you to access Python documentation. If you
+are completely new to Python, you can read the tutorial introduction
+in that document.
+
+There are many other editors and IDEs available, see :ref:`editors`
+for more information.
 
-If you want to run Python scripts from the Terminal window command line or from
-the Finder you first need an editor to create your script. macOS comes with a
-number of standard Unix command line editors, :program:`vim`
-:program:`nano` among them. If you want a more Mac-like editor,
-:program:`BBEdit` from Bare Bones Software (see
-https://www.barebones.com/products/bbedit/index.html) are good choices, as is
-:program:`TextMate` (see https://macromates.com). Other editors include
-:program:`MacVim` (https://macvim.org) and :program:`Aquamacs`
-(https://aquamacs.org).
+To run a Python script file from the terminal window, you can
+invoke the interpreter with the name of the script file:
 
-To run your script from the Terminal window you must make sure that
-:file:`/usr/local/bin` is in your shell search path.
+    |python_x_dot_y_literal| ``myscript.py``
```

Comment:
Unfortunately, this renders as `python3.14` `myscript.py` rather than `python3.14 myscript.py`.  I'm not sure whether fixing that is worth losing the easy replacement, unless maybe a double-colon and losing the backticks would work?

Reply:
I am not happy with how to handle versions properly but, for now, I would like to stick with some usage of references. I have changed all the 3.12 and 3.14 references to 3.13 for this PR. There's not much point in optimizing for 3.14 pre-alpha as this will all likely change by the time 3.14 releases. And we can worry about a possible backport to 3.12 later.